### DISCUSSION TITLE
Turn off continuous play ?

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#332] [Fixed Issues] [answered]
I have been using rAudio-1 since it was released (and Rune in many forms  before that) and I am currently using rAudio-1 20210515 (with Update Branch 20210518 applied to cure recent album playlist problem) on 5.10.25-4-ARCH aarch64. 

I have never noticed this before, but when playing albums (and web radios for that matter) the Player seems to be set to both Continuous play and Shuffle without me actually setting either.  I can see in the Playlist that I can choose Shuffle (but I never have) and I guess I could turn it off there if necessary but it doesn't seem to have actually shuffled my tracks. However I can't see any way to turn off Continuous play. 

Can anybody tell me how to do this please ?

Many thanks

Chris

**Comment by ceejayemm** [comment, marked as answer]
Answered my own question. I generally use the 'old' interface with the top and bottom bars. I found the Shuffle on/off in the Playlist screen but it only seems to affect the items in the Playlist. As noted above I was unable to find a Continuous Play setting.

It seems that the Shuffle and Continuous Play options that I needed are only available via the 'new' centre artwork touch options. When you touch the artwork display the new settings matrix revealed there has the Shuffle bottom left and Continuous Play bottom right. I have now set these options as I require. 

I much prefer the old style menu bars, particularly on a touchscreen as my 'big' fingers invariably end up pressing the wrong option.

Regards

Chris

**Reply by R101G** [reply]
I recall having exactly the same problem a while ago. 

@rern Perhaps the Buttons option in playback settings should include the same buttons under the cover art?
